src/main/java/chess/domain/board/BoardFactory.java

```diff
@@ -5,11 +5,33 @@
 import chess.domain.point.File;
 import chess.domain.point.Point;
 import chess.domain.point.Rank;
+import chess.dto.MovementDto;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
 
 public class BoardFactory {
 
+    private static final int FILE_INDEX = 0;
+    private static final int RANK_INDEX = 1;
+
+    public static Board createChessBoard(List<MovementDto> movementDtos) {
+        Board board = createInitialChessBoard();
+
+        if (movementDtos.isEmpty()) {
+            return board;
+        }
+
+        for (MovementDto movementDto : movementDtos) {
+            Point source = Point.of(File.of(movementDto.source().charAt(FILE_INDEX)),
+                    Rank.of(Integer.parseInt(String.valueOf(movementDto.source().charAt(RANK_INDEX)))));
+            Point target = Point.of(File.of(movementDto.target().charAt(FILE_INDEX)),
+                    Rank.of(Integer.parseInt(String.valueOf(movementDto.target().charAt(RANK_INDEX)))));
+            board.move(source, target);
+        }
+        return board;
```

Comment:
가독성이 너무 떨어져요. auto code formatting에만 의존하지 말고 적절한 개행을 통해 가독성을 도모해보세요.
```suggestion
    for (MovementDto movementDto : movementDtos) {
      Point source = Point.of(
          File.of(movementDto.source().charAt(FILE_INDEX)),
          Rank.of(Integer.parseInt(String.valueOf(movementDto.source().charAt(RANK_INDEX))))
      );
      Point target = Point.of(
          File.of(movementDto.target().charAt(FILE_INDEX)),
          Rank.of(Integer.parseInt(String.valueOf(movementDto.target().charAt(RANK_INDEX))))
      );
      board.move(source, target);
    }
```
엔터 4번만 해줘도 코드가 훨씬 이뻐지는 것 같네요.

Reply:
이 부분 해결하려고 Point 팩토리 메서드를 추가했었는데 적용하도록 하겠습니다. 말씀하신대로 auto code formatting 에 너무 의존하면 이런 일이 일어나네요.